src/test/java/github/weichware10/toolbox/zoommaps/ZoomTutorialTest.java

```diff
@@ -0,0 +1,30 @@
+package github.weichware10.toolbox.zoommaps;
+
+import github.weichware10.util.config.ConfigClient;
+import org.junit.Ignore;
+import org.junit.Test;
+
+
+/**
+ * Testet ZoomTutorial.
+ */
+public class ZoomTutorialTest {
+
+    /**
+     * Testen, ob die Einstellungen des configClient in Betracht genommen wird.
+     * D.h. ob das Tutorial angezeigt wird.
+     */
+    @Test
+    @Ignore
+    public void tutorialBooleanShouldBeRespected() {
+        ConfigClient configClient = new ConfigClient();
+        configClient.loadConfiguration("www.weichware10.com/config");
+        ZoomTutorial testsubject = new ZoomTutorial(configClient);
+        configClient.getConfig().getCodeChartsConfiguration().setTutorial(true);
```

Comment:
```suggestion
        configClient.getConfig().getZoomMapsConfiguration().setTutorial(true);
```
müsste das nicht so sein oder irre ich mich?

Reply:
Da hast du Recht

Reply:
![image](https://user-images.githubusercontent.com/82680013/144135324-53888383-dca7-4597-b446-c319b1f38ede.png)

Reply:
yikes lol